db/migrate/20230811110946_add_unique_index_to_orders.rb

```diff
@@ -0,0 +1,19 @@
+class AddUniqueIndexToOrders < ActiveRecord::Migration[7.0]
+  def change
+    reversible do |dir|
```

Comment:
I think this can be simplified by using `add_index` instead of explicit SQL queries